lib/images.sh

```diff
@@ -6,8 +6,11 @@ if [[ "${IMAGE_OS}" == "Ubuntu" ]]; then
   export IMAGE_NAME=${IMAGE_NAME:-UBUNTU_20.04_NODE_IMAGE_K8S_v1.19.3.qcow2}
   export IMAGE_LOCATION=${IMAGE_LOCATION:-https://artifactory.nordix.org/artifactory/airship/images/k8s_v1.19.3/}
 elif [[ "${IMAGE_OS}" == "FCOS" ]]; then
-  export IMAGE_NAME=${IMAGE_NAME:-fedora-coreos-32.20200923.2.0-openstack.x86_64.qcow2.xz}
-  export IMAGE_LOCATION=${IMAGE_LOCATION:-https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/32.20200923.2.0/x86_64/}
+  export IMAGE_NAME=${IMAGE_NAME:-fedora-coreos-32.20201104.3.0-openstack.x86_64.qcow2.xz}
+  export IMAGE_LOCATION=${IMAGE_LOCATION:-https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/32.20201104.3.0/x86_64/}
+elif [[ "${IMAGE_OS}" == "FCOS-ISO" ]]; then
+  export IMAGE_NAME=${IMAGE_NAME:-fedora-coreos-33.20201201.2.1-live.x86_64.iso}
+  export IMAGE_LOCATION=${IMAGE_LOCATION:-https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/33.20201201.2.1/x86_64}
```

Comment:
```suggestion
  export IMAGE_LOCATION=${IMAGE_LOCATION:-https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/33.20201201.2.1/x86_64/}
```

Reply:
@fmuyassarov thanks for the suggestion, but I don't think the trailing slash is needed:

```
./01_prepare_host.sh:    curl --insecure --compressed -O -L "${IMAGE_LOCATION}/${IMAGE_NAME}"
```

It's also used inconsistently, so we should perhaps consider removing it for existing images in a separate PR?

Reply:
You are right.  
+1 for removing extra slash in another PR.